### ISSUE TITLE
ComboBox: Can't open dropdown options with VoiceOver/TalkBack turned on 

### ISSUE THREAD

**Issue by erichdev** [microsoft/fluentui#14370]
### Environment Information

- **Package version(s)**: Latest/7.125
- **Browser and OS versions**: Chrome 84/Android 10, iOS 13.5.1

### Describe the issue:

In mobile with TalkBack or VoiceOver turned on, the ComboBox dropdown options cannot be opened. This does not repro with TalkBack or VoiceOver turned off.

### Please provide a reproduction of the issue in a codepen:

This reproes on the docs site ComboBox example: https://fabricweb.z5.web.core.windows.net/oufr/7.125.0/#/examples/ComboBox

1. In mobile, turn on VoiceOver or TalkBack
2. Tap the first ComboBox example
3. Double tap to open

#### Actual behavior:

The keyboard opens, but the dropdown options never open.

#### Expected behavior:

The dropdown options open when double tapping the ComboBox.

w3.org has a working combobox example where the dropdown options opens with TalkBack turned on (example 3): https://www.w3.org/TR/wai-aria-practices/examples/combobox/aria1.1pattern/listbox-combo.html

I think a potential solution is to make the caret focusable, but not sure if that is the right approach. I created a minimal repro here: https://codepen.io/erichdev/pen/jOqOeMq
Clicking the button "Make caret focusable" will set `tabindex=0` on the dropdown caret. The caret is then clickable, although it requires two double taps to open the dropdown.

**Comment by erichdev** [comment]
Setting `isButtonAriaHidden=false` mostly resolves this issue. This allows users to swipe to focus on the dropdown caret to open the options. However, two double taps are required to open the options -- the first double tap puts focus back on the input and dismisses the options. 

Also, should `isButtonAriaHidden` be set to `false` by default to support mobile users with screenreaders?

**Comment by khmakoto** [comment]
@smhigley what do you think of @erichdev's suggestion?
